src/commonMain/kotlin/com/sunnychung/lib/multiplatform/kdatetime/KDateTimeFormat.kt

```diff
@@ -198,8 +210,10 @@ class KDateTimeFormat(val pattern: String) {
 
     protected fun parseAmPm(input: String): Int {
         return when (input.lowercase()) {
```

Comment:
As caps and non-caps names are introduced, the process of converting the input to lowercase should be removed, otherwise the caps names never match.

Reply:
Agree. Changed `input.lowercase()` into `input`

Reply:
Would you please change `input.lowercase()` to `input`? Otherwise if the input is in uppercase, it can be mapped to lowercase one, which should throw error instead.